src/types/state.ts

```diff
@@ -182,6 +198,8 @@ const DeviceReport = t.partial({
 	cpu_usage: t.number,
 	cpu_id: t.string,
 	is_undervolted: t.boolean,
+	// These are for internal reporting only, they are not sent
+	// to the API
 	update_failed: t.boolean,
 	update_pending: t.boolean,
 	update_downloaded: t.boolean,
```

Comment:
Since we do omit these fields in `getCurrentForReport`, perhaps we can remove some code that still attempts to report these fields: [[1]](https://github.com/balena-os/balena-supervisor/blob/45d1f1f964e777b5168b089858af0cec33f98c68/src/device-state/index.ts#L238-L240) [[2]](https://github.com/balena-os/balena-supervisor/blob/45d1f1f964e777b5168b089858af0cec33f98c68/src/device-state/index.ts#L536) [[3]](https://github.com/balena-os/balena-supervisor/blob/45d1f1f964e777b5168b089858af0cec33f98c68/src/device-state/index.ts#L632-L636)

However, good to save for another PR